### ISSUE TITLE
local data example not working

### ISSUE THREAD

**Issue by alexvpickering** [gosling-lang/gos#158]
I am trying to run the example for using local data from [here](https://gosling-lang.github.io/gos/user_guide/local_data.html) using the [cistrom multivec file](https://s3.amazonaws.com/gosling-lang.org/data/cistrome.multires.mv5).

```bash
pip install gosling[all]
```

```python
import gosling as gos

data = gos.multivec(
    url='./cistrome.multires.mv5', # path to local multivec
    row="sample",
    column="position",
    value="peak",
    categories=["sample 1", "sample 2", "sample 3", "sample 4"],
    binSize=4,
)

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/alex/miniforge3/lib/python3.10/site-packages/gosling/data/__init__.py", line 116, in load
    url = data_server(data)
  File "/home/alex/miniforge3/lib/python3.10/site-packages/gosling/data/__init__.py", line 92, in __call__
    self._resources[resource_id] = self._provider.create(data, **kwargs)
  File "/home/alex/miniforge3/lib/python3.10/site-packages/servir/_provide.py", line 109, in create
    if resource.uid in self._tilesets:
  File "/home/alex/miniforge3/lib/python3.10/site-packages/servir/_tilesets.py", line 46, in uid
    return self._tileset.uid
AttributeError: 'Tileset' object has no attribute 'uid'
```

Not sure if this is a bug or something I'm doing wrong. Thanks!

**Comment by manzt** [comment]
Hi @alexvpickering,

I think this was a regression when we migrated to servir for a data source. I think I have a fix in #159

**Comment by manzt** [comment]
I just published a release with the fix in #159. I can confirm that the example from the docs works as expected in Jupyter.

**Comment by alexvpickering** [comment]
Hi @manzt - thanks for the quick fix!
